app/kumactl/cmd/install/render_helm_files.go

```diff
@@ -17,6 +17,28 @@ import (
 	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
 )
 
+type fileFilter func(data.File) bool
+
+var helmTemplateFilters []fileFilter
+
+func filterHelmTemplates(files data.FileList) data.FileList {
```

Comment:
What if you updated the InstallCrdTemplateFiles hook on the InstallCrdsContext, and the InstallCpTemplateFiles hook on the other one?

Reply:
This would require updating it anywhere else a different `Install*Context` is used.